src/types/ActionObject.ts

```diff
@@ -196,17 +196,13 @@ export type FetchAction = {
     allTrainDialogs: CLM.TrainDialog[],
 } | {
     type: AT.FETCH_ACTIVITIES_ASYNC,
-    appId: string,
-    userName: string,
-    userId: string,
-    trainDialog: CLM.TrainDialog
+    noSpinner: boolean
```

Comment:
Does this mean `displaySpinner` or does `no` mean something else besides display?

Reply:
Means show no spinner.  Can rename it to "noSpinnerDisplay"

Reply:
Seems like a weird variable name to me. It allows double negatives and those should be avoided.

`displaySpinner: true` Display spinner
`displaySpinner: false` Don't display spinner
`noSpinnerDisplay: false` : ? Does that mean spinner is displayed because it was no display?

Reply:
Generally I lean towards making the uncommon condition (the exception / added feature) to true.   but I'm happy to reverse it